ODLA/platforms/ascend/src/CMakeLists.txt

```diff
@@ -0,0 +1,46 @@
+# ==============================================================================
+# Copyright (C) 2019-2020 Alibaba Group Holding Limited.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+# ==============================================================================
+cmake_minimum_required(VERSION 3.14)
+
+project(olda_ascend_lib)
+
+set(CMAKE_SKIP_RPATH TRUE)
+set(ASCEND_PATH "/usr/local/Ascend")
+set(ODLA_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/../../../)
+set(CMAKE_CXX_FLAGS "-std=c++11 -g -Wall -D_GLIBCXX_USE_CXX11_ABI=0")
+
+include_directories(
+    ${ASCEND_PATH}/ascend-toolkit/latest/opp/op_proto/built-in/inc
+    ${ASCEND_PATH}/ascend-toolkit/latest/fwkacllib/include/graph
+    ${ASCEND_PATH}/ascend-toolkit/latest/fwkacllib/include/ge
+    ${ASCEND_PATH}/ascend-toolkit/latest/fwkacllib/include/
+    ${ASCEND_PATH}/ascend-toolkit/latest/fwkacllib/include/parser
+    ../inc
+    ${ODLA_ROOT}/include
+)
+
+link_directories(
+    ${ASCEND_PATH}/ascend-toolkit/latest/opp/op_impl/built-in/ai_core/tbe/op_tiling/
+    ${ASCEND_PATH}/ascend-toolkit/latest/fwkacllib/lib64/plugin/opskernel/
+    ${ASCEND_PATH}/ascend-toolkit/latest/fwkacllib/lib64/stub/
+)
+
+add_library(odla_ascend SHARED model_process.cc odla_ascend_acl.cc odla_ops.cc)
```

Comment:
we use add_odla_library(odla_asend model_process.cc odla_asend_acl.cc odla_ops.cc)